specification/streamanalytics/resource-manager/Microsoft.StreamAnalytics/preview/2017-04-01-preview/functions.json

```diff
@@ -0,0 +1,964 @@
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.StreamAnalytics/streamingjobs/{jobName}/functions/{functionName}": {
+      "put": {
+        "tags": [
+          "Functions"
+        ],
+        "operationId": "Functions_CreateOrReplace",
+        "description": "Creates a function or replaces an already existing function under an existing streaming job.",
+        "x-ms-examples": {
+          "Create an Azure ML function": {
+            "$ref": "./examples/Function_Create_AzureML.json"
+          },
+          "Create a JavaScript function": {
+            "$ref": "./examples/Function_Create_JavaScript.json"
+          }
+        },
+        "parameters": [
+          {
+            "name": "function",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Function"
+            },
+            "description": "The definition of the function that will be used to create a new function or replace the existing one under the streaming job."
+          },
+          {
+            "name": "If-Match",
+            "in": "header",
+            "required": false,
+            "type": "string",
+            "x-ms-client-name": "IfMatch",
+            "description": "The ETag of the function. Omit this value to always overwrite the current function. Specify the last-seen ETag value to prevent accidentally overwriting concurrent changes."
+          },
+          {
+            "name": "If-None-Match",
+            "in": "header",
+            "required": false,
+            "type": "string",
+            "x-ms-client-name": "IfNoneMatch",
+            "description": "Set to '*' to allow a new function to be created, but to prevent updating an existing function. Other values will result in a 412 Pre-condition Failed response."
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/StreamingJobNameParameter"
+          },
+          {
+            "$ref": "#/parameters/FunctionNameParameter"
+          }
+        ],
+        "responses": {
```

Comment:
>responses [](start = 9, length = 9)

You need to declare a default error response in all the operations in all the swaggers in this PR.

Reply:
Do you mean that we need to add a default error response like:
```
        "responses": {
          "200": {

          },
          "202": {

          },
          "default": {
            "description": "Error.",
            "schema": {
              "$ref": "../../common/v1/definitions.json#/definitions/Error"
            }
          }
        },
```


https://github.com/Azure/azure-rest-api-specs/blob/aa91867b7b83be0534ce3feb2583c12e0cdd2bff/specification/streamanalytics/resource-manager/Microsoft.StreamAnalytics/preview/2020-03-01-preview/clusters.json#L90